examples/04_glm_first_level/plot_hrf.py

```diff
@@ -1,7 +1,7 @@
-"""Example of hemodynamic reponse functions.
+"""Example of hemodynamic response functions.
 =========================================
```

Comment:
```suggestion
"""Example of hemodynamic response functions.
==========================================
```

Reply:
yes pep8 does break the CI. It makes the PRs red on merge. There is pep8 action that we cannot ignore.

Regarding the RST fix with the missing `=` it is RestructedText syntax that enforces that section subtitles have the right number of = or - or ^ etc.

Reply:
I don't see how I break the CI. The line of code hasn't changed, except for a fixed typo **and** less characters per line than before. The issue raised by the CI precedes my patch, so it cannot possibly break the CI more than the previous code. What do you mean by "we cannot ignore", as far as I know it's entirely possible to bypass the CI if needed when merging.

Reply:
Indeed, the [reStructedText specification](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections) states:
> Sections are identified through their titles, which are marked up with adornment: "underlines" below the title text, or underlines and matching "overlines" above the title. An underline/overline is a single repeated punctuation character that begins in column 1 and forms a line extending at least as far as the right edge of the title text.

But then I'm not certain what "_extending at least as far as the right edge of the title text_" means. Exact number of characters or visible length in any editor with a possibly variable-width font? I'm asking because while only the first interpretation makes sense, the rule is (almost) never enforced in nibabel. Actually, the suggested fix doesn't fix anything, the underline was too short before the commit and remains too short after the commit.

Reply:
yes it must be at least the number of characters. If it's too short please fix it. Thanks a lot @DimitriPapadopoulos

Reply:
> Actually, the suggested fix doesn't fix anything, the underline was too short before the commit and remains too short after the commit.


the spell fix did add a character to the title, making the underline, which _was_ the right length before, too short. the fix proposed by @agramfort does fix the issue. Note the `"""` that enclose a python literal string are not part of the docstring itself. I understand the frustration, rst syntax can be quite annoying and the error messages are not always helpful.

Reply:
Indeed, but indentation doesn't help, I would change:
```
"""Example of hemodynamic reponse functions.
=========================================
```
to:
```
"""
Example of hemodynamic reponse functions.
=========================================
```

And reST-like incosistent syntax in non-reST text doesn't help either:
```
############################################################################
# Fetching probabilistic atlas - MSDL atlas
# -----------------------------------------
[...]
#############################################################################
# Visualizing a probabilistic atlas with plot_stat_map and add_overlay object
# ---------------------------------------------------------------------------

[...]
###############################################################################
# Visualizing a probablistic atlas with plot_prob_atlas
# =====================================================
```

Reply:
Anyway, the CI errors have been fixed.